lib/elements/stroke.py

```diff
@@ -472,7 +472,12 @@ def to_stitch_groups(self, last_patch):
                 path = [Point(x, y) for x, y in path]
                 # manual stitch
                 if self.manual_stitch_mode:
-                    patch = StitchGroup(color=self.color, stitches=path, lock_stitches=(None, None))
+                    patch = StitchGroup(color=self.color,
+                                        stitches=path,
+                                        lock_stitches=self.lock_stitches,
+                                        trim_after=self.trim_after,
+                                        stop_after=self.stop_after,
```

Comment:
These aren't necessary, and trim/stop do currently work with manual stitch.  `self.trim_after` and `self.stop_after` are checked in `EmbroideryElement.embroider()`, and the visual command versions of trim/stop are also checked at that point, so checking here is superfluous.